tests/test_downloader.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+from downloader.download_manager import DownloadManager
+from downloader.history_downloader.history_manager import HistoryDownloader
+from downloader.history_saver.save_manager import HistorySaver
+
+
+class DownloaderTest(unittest.TestCase):
+
+    def test_downloaded_data_type(self):
+        history_downloader = HistoryDownloader('binance', 'BTC/USDT', 4)
+        self.assertTrue(type(history_downloader.get_history()) == list,

```

Comment:
You should have more specific assert available in Unittest framework. eg. self.assertIsInstance()

Reply:
Sure, this method is more specific. Changed it.